src/content/cloud/personal-access-tokens.mdx

```diff
@@ -26,9 +26,9 @@ Personal Access Tokens are useful when scripting, or when configuring [Preview E
 
 ### Token Expiration
 
-Personal Access Tokens are valid for 180 days. When a token has expired, it can no longer be used to authenticate and for API requests. It is not possible to restore an expired token, you will need to create a new token upon expiration.
+When you create a token, you can select the expiration, by default is 180 days. Once a token has expired, it can no longer be used to authenticate and for API requests. It is not possible to restore an expired token, you will need to create a new token upon expiration.
```

Comment:
```suggestion
When you create a token, you can select the expiration, the default expiration is 180 days. Once a token has expired, it can no longer be used to authenticate or for API requests. It is not possible to restore an expired token, you will need to create a new token upon expiration.
```